source/documentation/getting_started/quick_setup_guide.md

```diff
@@ -42,7 +42,7 @@ GOV.UK PaaS uses a hosting technology called Cloud Foundry. As a tenant (that is
 
     Your `USERNAME` is your email address your account was created with.
 
-    You will then be prompted to enter your password, which should have been sent to you by email (subject "Welcome to GOV.UK PaaS").
+    You will then be prompted to enter your password, which you should have set from the invite link in your welcome email (subject "Welcome to GOV.UK PaaS").
```

Comment:
Suggested simpler wording. 

You will then be prompted to enter your password. You set this when you clicked on the invite link in your welcome email